pkg/ffuf/job.go

```diff
@@ -107,10 +108,22 @@ func (j *Job) Start() {
 		j.startTime = time.Now()
 	}
 
-	// Add the default job to job queue
-	j.queuejobs = append(j.queuejobs, QueueJob{Url: j.Config.Url, depth: 0})
+	basereq := BaseRequest(j.Config)
+
+	if j.Config.InputMode == "sniper" {
+		// process multiple payload locations and create a queue job for each location
+		reqs := SniperRequests(&basereq, "§")
```

Comment:
I assume the template string should be fetched from the `InputProvider` where we set it up in `optionsparser.go`, making it easier to change later on or maybe even added as user configurable value in the future.

Reply:
Good catch! Should be an easy fix